railties/lib/rails.rb

```diff
@@ -67,7 +67,7 @@ def root
     #   Rails.env.development? # => true
     #   Rails.env.production? # => false
     def env
-      @_env ||= ActiveSupport::StringInquirer.new(ENV["RAILS_ENV"] || ENV["RACK_ENV"] || "development")
+      @_env ||= ActiveSupport::StringInquirer.new(ENV["RAILS_ENV"].presence || ENV["RACK_ENV"].presence || "development")
```

Comment:
Thanks @rafaelfranca . Nice catch! I've required the file now, please see https://github.com/rails/rails/pull/27399/commits/a1c0ccab8c65c33585458143a7bda6b55c43b4d7 . Cheers!